extensions/keyboard-layout-switcher/src/switch.tsx

```diff
@@ -33,7 +33,7 @@ function toastErrorOptions(message: string, error: string): Toast.Options {
 async function handleLayoutSwitch(source: ILayout) {
   try {
     await source.activate();
-    await Promise.all([showHUD(`Activated ${source.title} Layout`), popToRoot(), closeMainWindow()]);
+    await Promise.all([showHUD(`⌨️ Activated ${source.title} Layout`), popToRoot(), closeMainWindow()]);
```

Comment:
Uff it has been a while since I wrote this. But I would actually add the `'` around the layout name to _separate_ the name from the actual message.

```suggestion
  await showHUD(`⌨️ Activated '${source.title}' Layout`);
```

Reply:
Done it, to both HUD @lucaschultz